### PR TITLE
ibmcloud: add skip_verify_console variable

### PR DESCRIPTION
**Pull request by GeorginaKin**
Allows skipping of console output during verify

Fixes: #89
Signed-off-by: Georgina Kinge <georgina.kinge@ibm.com>

### CODE REVIEW

File: ibmcloud/image/verify.sh [hunk 1]
```diff
@@ -54,6 +54,10 @@ while ! ibmcloud is instance $id --output json | jq -e '.status == "running"' >
     sleep 5
 done
 
+skip_verify_console=$SKIP_VERIFY_CONSOLE
+
+if ["$skip_verify_console" = false] ; then
+
```
Comment (yoheiueda) [inline ibmcloud/image/verify.sh:60]: Could you change the code to skip console checking only when the variable is set?

The code will be something like this.

```
if [[ -n "${SKIP_VERIFY_CONSOLE:-}" ]]; then
	echo 'SKIP_VERIFY_CONSOLE is set. skip console checking'
	exit 0
fi
```

`"${SKIP_VERIFY_CONSOLE:-}"` will suppress the "unbound variable" error, even when an older version of bash is used.

File: ibmcloud/image/verify.sh [hunk 2]
```diff
@@ -99,3 +103,4 @@ os.write(fd, b"\035\n") # send ^[
 os.write(1, b"\n")
 sys.exit(exit_status)
 END
+fi
```
Comment (yoheiueda) [inline ibmcloud/image/verify.sh:106]: Then, remove this line.

**Review by yoheiueda — CHANGES_REQUESTED** [review]
Thank you very much for contributing this patch!

File: ibmcloud/image/verify.sh (hunk 2 above)
Comment (GeorginaKin) [reply ibmcloud/image/verify.sh]: Thank you so much for reviewing these changes! I've made some further changes based on your comments :)

File: ibmcloud/image/verify.sh [hunk 3]
```diff
@@ -54,6 +54,13 @@ while ! ibmcloud is instance $id --output json | jq -e '.status == "running"' >
     sleep 5
 done
 
+skip_verify_console=$SKIP_VERIFY_CONSOLE
+
```
Comment (yoheiueda) [inline ibmcloud/image/verify.sh:58]: This line is no longer necessary.

**Review by yoheiueda — CHANGES_REQUESTED** [review]

File: ibmcloud/image/verify.sh (hunk 3 above)
Comment (GeorginaKin) [reply ibmcloud/image/verify.sh]: Ah because we won't have the unbound variable issue now of course - I've taken out this line too, thanks!

**Review by yoheiueda — APPROVED** [review]
LGTM. Thank you very much!

**Pull request merged**